test/main.go

```diff
@@ -0,0 +1,51 @@
+package test
+
+import (
+	"errors"
+	"log"
+	"math/rand"
+	"os"
+	"os/exec"
+)
+
+const (
+	letterBytes   = "abcdefghijklmnopqrstuvwxyz"
+	randSuffixLen = 8
+)
+
+func RandomString() string {
```

Comment:
I don't see where this function gets used

Reply:
Sorry, I pushed it to show it you, but showed you my editor instead. The PR is a draft, I'm still considering some options when it comes to integration testing.

Reply:
Now its being used.